### ISSUE TITLE
Dota FPS Drop When Moving Mouse Cursor Onto Ultimate Ability

### ISSUE THREAD

**Issue by Cthulhu001** [ValveSoftware/Dota2-Gameplay#15381]
### Description

When I move my mouse cursor onto the ultimate ability of any heroes, the game lags instantly but when I move it on somewhere else or other abilities, it doesn't be a problem.  Already verified and reinstall my graphic driver but has not been solved. How can I solve it?

**Comment by IgorHoruzha** [comment]
Do You still has the issue?

**Comment by Cthulhu001** [comment]
Yes, FPS is lowing down to 5 for a sec, then it's coming back. If I move it down and up on the ultimate, it becomes unplayable.

**Comment by Langdris** [comment]
Encountered same issue, also when you hover your mouse into Aghanim's Scepter's description, my FPS goes from 120 to 20-30fps for a second.

**Comment by tuannguyen0111** [comment]
Encountered same issue, fixed it by change to defaut terrain.

**Comment by AKvint** [comment]
Try to use the default guide from dota 2 for heroes.

**Comment by sergolius** [comment]
Described root cause: https://github.com/ValveSoftware/Dota2-Gameplay/issues/17566

Simple solution is to not use "hero guides" with "Ability Guide"
